index.html

```diff
@@ -1703,11 +1703,54 @@
 
       <section><h3>Obtaining push permission</h3>
       <p>
-        To <dfn>obtain push permission</dfn>, run these steps:
+        To <dfn>obtain push permission</dfn> given <var>target</var> of type
+        <code><a>NFCPushTarget</a></code>, run these steps:
         <ol>
           <li>
             If a <a>prearranged trust relationship</a> exists,
-            then return <code>true</code>.
+            return <code>true</code>.
+          </li>
+          <li>
+            If <var>target</var> has the value <code>"peer"</code>, i.e. the
+            push is requested on an <a>NFC peer</a>, return <code>true</code>.
+            <p class="note">
+              The need for explicit user gesture for tapping an <a>NFC peer</a>,
+              together with the other <a href="#security-policies">
```

Comment:
The #security-policies section is non-normative (and can't be normative because it's not precise enough), so this section can't rely on it for security properties. This section instead needs to spell out how to achieve the policies from that section.